### PR TITLE
TM-5575: Popup implementation examples

### PR DESCRIPTION
**Pull request by musialp**
Add code of Custom popup examples to Tag Manager space

### CODE REVIEW

File: tag_manager/custom_popup_examples.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Since version 10.1 of PPAS there is a possiblity of creating a `Custom popup` tag template. We decided to share some example implementations that can be further modified and expanded.
+
+Example 1
+----------
+Preview:
+    .. image:: /_static/images/tm_popup_examples/example-popup-1.png
+        :alt: Custom popup example 1
+
+.. note::
+    Handling of the close button is provided in ContainerJS, as long as the class name ``ppms-popup-close-button`` is unchanged. JavaScript code to handle `Subscribe now` button needs to be provided.
```
Comment (zawias-pro) [inline tag_manager/custom_popup_examples.rst]: not sure if clients are aware of ContainerJS keyword

Comment (zawias-pro) [inline tag_manager/custom_popup_examples.rst]:
```suggestion
    Handling of the close button is provided in ContainerJS, as long as the class name ``ppms-popup-close-button`` is unchanged. Your own JavaScript code to handle `Subscribe now` button needs to be provided.
```

File: tag_manager/popup_examples/example-popup-1.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    line-height: 18px;
+    margin-top: 24px;
+  }
+
+  .ppms-popup-input {
+    display: block;
+    width: 100%;
+    box-sizing: border-box;
+    height: 36px;
+  	border: 1px solid #999999;
```
Comment (zawias-pro) [inline tag_manager/popup_examples/example-popup-1.html]:
```suggestion
    border: 1px solid #999999;
```

**Comment by zawias-pro** [comment]
Example 2
image

File: tag_manager/custom_popup_examples.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+.. default-domain:: js
+
+.. versionadded:: 10.1
+
+Custom popup template implementation examples
+=============================================
+
+Introduction
+------------
+Since version 10.1 of PPAS there is a possiblity of creating a `Custom popup` tag template. We decided to share some example implementations that can be further modified and expanded.
```
Comment (zawias-pro) [inline tag_manager/custom_popup_examples.rst]: This is too brief I think. There should be some instructions about how to add Custom Popup tag.

File: tag_manager/custom_popup_examples.rst (hunk 1 above)
Comment (musialp) [reply tag_manager/custom_popup_examples.rst]: Change to a more general `provided out of the box`

**Pull request merged**